### ISSUE TITLE
codemirror in w2ui layout causes gutter display issue.

### ISSUE THREAD

**Issue by andrewpeterson** [codemirror/codemirror5#1563]
w2ui is a jQuery layout plugin, which allows you to load content from separate HTML pages into an overall layout. When a codemirror instance is included in one of these layouts, the gutter does not extend the entire height of the codemirror control. See image below:

![codemirror-bug](https://f.cloud.github.com/assets/4583291/593495/cb4a5ff2-ca39-11e2-92bb-fb2bb1a533f1.jpg)

This has been tested with codemirror-3.13 and the latest developer version from git. It has been confirmed to occur with Firefox 21 and IE9 on Windows 7, and Firefox 21 on Fedora 17.

The example is trivial, using the code below. It requires that **codemirror** is installed to <code>/codemirror</code> and **w2ui** is installed to <code>/w2ui</code>. **You need to press enter to get the problem to occur**

<pre><code>
&lt;!DOCTYPE html PUBLIC &quot;-//W3C//DTD HTML 4.01 Transitional//EN&quot; &quot;http://www.w3.org/TR/html4/loose.dtd&quot;&gt;
&lt;html&gt;
&lt;head&gt;
&lt;meta http-equiv=&quot;Content-Type&quot; content=&quot;text/html; charset=ISO-8859-1&quot;&gt;
&lt;title&gt;ui&lt;/title&gt;

&lt;script src=&quot;//ajax.googleapis.com/ajax/libs/jquery/1.10.0/jquery.min.js&quot;&gt;&lt;/script&gt;

&lt;link rel=&quot;stylesheet&quot; type=&quot;text/css&quot; href=&quot;w2ui/w2ui-1.2.min.css&quot; /&gt;
&lt;script type=&quot;text/javascript&quot; src=&quot;w2ui/w2ui-1.2.min.js&quot;&gt;&lt;/script&gt;

&lt;link rel=&quot;stylesheet&quot; href=&quot;codemirror/lib/codemirror.css&quot;&gt;
&lt;script src=&quot;codemirror/lib/codemirror.js&quot;&gt;&lt;/script&gt;
&lt;link rel=&quot;stylesheet&quot; href=&quot;codemirror/theme/ambiance.css&quot;&gt;
&lt;script src=&quot;codemirror/mode/javascript/javascript.js&quot;&gt;&lt;/script&gt;

&lt;/head&gt;

&lt;body&gt;
    &lt;div id=&quot;layout&quot; style=&quot;width: 95%; height: 1500px&quot;&gt;&lt;/div&gt;
    &lt;div id=&quot;vince&quot; style=&quot;width: 600px&quot;&gt;
        &lt;textarea id=&quot;code&quot; name=&quot;code&quot;&gt;some code&lt;/textarea&gt;
    &lt;/div&gt;
&lt;/body&gt;

&lt;script&gt;
  var editor = CodeMirror.fromTextArea(document.getElementById(&quot;code&quot;), {
    lineNumbers: true
  });
  editor.setOption(&quot;theme&quot;, &quot;ambiance&quot;);

    var pstyle = 'border: 1px solid #dfdfdf; padding: 5px;'
    $('#layout').w2layout({
      name: 'layout',
      panels: [
        { type: 'top', size: 50, style: pstyle, content: 'top', resizable: true },
        { type: 'left', size: 300, style: pstyle, content: 'left', resizable: true },
        { type: 'main', style: pstyle }
      ]
    });
    w2ui['layout'].content('main', document.getElementById(&quot;vince&quot;));

&lt;/script&gt;
&lt;/html&gt;
</code></pre>
